WordPress/Sniffs/CodeAnalysis/PerformantPHPFunctionsSniff.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WordPressCS\WordPress\Sniffs\CodeAnalysis;
+
+use WordPressCS\WordPress\AbstractFunctionRestrictionsSniff;
+
+/**
+ * Encourages the use of more performant PHP.
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since 3.0.0
+ */
+class PerformantPHPFunctionsSniff extends AbstractFunctionRestrictionsSniff {
```

Comment:
This name seems to imply that more groups will be added at a later point. Do you have any other ones mind ?

Also, the suggestion is to *not* use these functions, but use more performant alternatives, while the name of the sniff gives the impression that the functions would be more performant.

Altogether, might be good to have another think about the name.

And yes, naming things is hard ;-)